doc/letsencrypt.md

```diff
@@ -0,0 +1,133 @@
+# Terminal with a Let's Encrypt Certificate
+
+You can configure the HTTPS server to automatically install a free SSL certificate
+provided by [LetsEncrypt](https://letsencrypt.org/). This is recommended if you plan to
+access the website from a remote computer and do not want to deal with the browser warning
+you about the self-signed certificate. Another benefit is you can use a memorable URL to
+access the Terminal UI instead of an IP address.
+
+> Note: LiT only serves content over **HTTPS**. If you do not use `letsencrypt`, LiT will
+> use the self-signed certificate that is auto-generated by `lnd` to encrypt the
```

Comment:
Worth adding "auto-generated by `lnd` (or LiT itself if `lnd` is running on a remote host)"?